🔒 Upgrade plugins for Jenkins security advisory 2019-03-25

grdryn opened the pull request.

grdryn commented:
I also noticed when I went to build using `Dockerfile.localdev`, that there's an error message, but it builds anyway:

> /usr/local/bin/install-plugins.sh: line 324: jar: command not found

I think that `jar` binary comes from `java-1.8.0-openjdk-devel`. Would you like me to add that to get installed in that Dockerfile here? I see it's already being installed in the Dockerfile.rhel7.

gabemontero commented:
Hey @grdryn 

A few items:

1) these look like the changes for https://jenkins.io/security/advisory/2019-03-25/ and not https://jenkins.io/security/advisory/2019-03-06/

So can you update the comment section accordingly

2) yeah I missed the missing `jar` for Dockerfile.localdev message ... absolutely, please add that update to this PR (separate commit is fine) if you don't mind :-)

3) we are hitting rpm mirroring problems on our CI cluster again ...  it may resolve on its own, but I'll be engaging folks in my ORG separately about this (again)

gabemontero commented:
/hold

(until api.ci rpm mirror returns to health)

gabemontero commented:
@smarterclayton has provided https://github.com/openshift/release/pull/3250 ... let's see how things go

gabemontero commented:
/hold cancel

gabemontero commented:
/retest

gabemontero commented:
this has not been working lately, but just in case 

/cherrypick openshift-3.11

grdryn commented:
> Hey @grdryn
> 
> A few items:
> 
>     1. these look like the changes for https://jenkins.io/security/advisory/2019-03-25/ and not https://jenkins.io/security/advisory/2019-03-06/
> 
> 
> UPDATE (I thought the comment was missing but I see it now) .... I updated the PR title.

Oops, sorry about that, I've updated the commit message now too.

> 
>     1. yeah I missed the missing `jar` for Dockerfile.localdev message ... absolutely, please add that update to this PR (separate commit is fine) if you don't mind :-)

Done! :+1: 

>     2. we are hitting rpm mirroring problems on our CI cluster again ...  it may resolve on its own, but I'll be engaging folks in my ORG separately about this (again)

gabemontero commented:
/approve

gabemontero commented:
/lgtm

gabemontero commented:
/retest

gabemontero commented:
/retest

gabemontero commented:
been working with a variety of folks ... the api.ci mirrors appear functional again

Installed plugins:
```
ace-editor:1.0.1
apache-httpcomponents-client-4-api:4.5.5-3.0
authentication-tokens:1.3
blueocean-autofavorite:1.2.2
blueocean-bitbucket-pipeline:1.10.2
blueocean-commons:1.10.2
blueocean-config:1.10.2
blueocean-core-js:1.10.2
blueocean-dashboard:1.10.2
blueocean-display-url:2.2.0
blueocean-events:1.10.2
blueocean-github-pipeline:1.10.2
blueocean-git-pipeline:1.10.2
blueocean-i18n:1.10.2
blueocean-jira:1.10.2
blueocean:1.10.2
blueocean-jwt:1.10.2
blueocean-personalization:1.10.2
blueocean-pipeline-api-impl:1.10.2
blueocean-pipeline-editor:1.10.2
blueocean-pipeline-scm-api:1.10.2
blueocean-rest-impl:1.10.2
blueocean-rest:1.10.2
blueocean-web:1.10.2
branch-api:2.0.20
cloudbees-bitbucket-branch-source:2.2.15
cloudbees-folder:6.6
conditional-buildstep:1.3.1
config-file-provider:3.5
credentials-binding:1.15
credentials:2.1.18
display-url-api:2.2.0
docker-commons:1.11
docker-workflow:1.14
durable-task:1.26
favorite:2.3.1
git-client:2.7.3
github-api:1.90
github-branch-source:2.3.6
github:1.29.2
github-organization-folder:1.6
git:3.9.3
git-server:1.7
handlebars:1.1
handy-uri-templates-2-api:2.1.6-1.0
htmlpublisher:1.16
jackson2-api:2.8.11.3
jenkins-design-language:1.10.2
jira:3.0.5
job-dsl:1.72
jquery-detached:1.2.1
jsch:0.1.54.1
junit:1.26.1
kubernetes-credentials:0.3.0
kubernetes:1.12.8
lockable-resources:2.5
mailer:1.21
mapdb-api:1.0.9.0
matrix-auth:2.2
matrix-project:1.14
mercurial:2.3
momentjs:1.1
openshift-client:1.0.29
openshift-login:1.0.16
openshift-sync:1.0.34
parameterized-trigger:2.35.2
pipeline-build-step:2.7
pipeline-github-lib:1.0
pipeline-graph-analysis:1.7
pipeline-input-step:2.8
pipeline-milestone-step:1.3.1
pipeline-model-api:1.3.4.1
pipeline-model-declarative-agent:1.1.1
pipeline-model-definition:1.3.4.1
pipeline-model-extensions:1.3.4.1
pipeline-rest-api:2.10
pipeline-stage-step:2.3
pipeline-stage-tags-metadata:1.3.4.1
pipeline-stage-view:2.10
pipeline-utility-steps:2.1.0
plain-credentials:1.4
pubsub-light:1.12
run-condition:0.10
scm-api:2.2.8
script-security:1.56
sse-gateway:1.16
ssh-credentials:1.14
structs:1.17
subversion:2.10.3
token-macro:2.6
variant:1.1
workflow-aggregator:2.6
workflow-api:2.32
workflow-basic-steps:2.11
workflow-cps-global-lib:2.11
workflow-cps:2.65
workflow-durable-task-step:2.26
workflow-job:2.29
workflow-multibranch:2.20
workflow-remote-loader:1.4
workflow-scm-step:2.6
workflow-step-api:2.19
workflow-support:3.2
```

gabemontero commented:
hitting https://bugzilla.redhat.com/show_bug.cgi?id=1691516 looking flakes ... getting triaged in #forum-installer as I type

gabemontero commented:
/retest

gabemontero commented:
/retest

gabemontero commented:
/retest

gabemontero commented:
rhel plugin rpm job https://buildvm.openshift.eng.bos.redhat.com:8443/job/devex/job/devex%252Fjenkins-plugins/114/

dist git commit http://pkgs.devel.redhat.com/cgit/rpms/jenkins-2-plugins/commit/?h=rhaos-4.0-rhel-7&id=714c6bb2673e349315bd1751078d55b38ce6ee87
